bot/exts/info/python_news.py

```diff
@@ -75,8 +75,17 @@ async def get_webhook_names(self) -> None:
 
     @staticmethod
     def escape_markdown(content: str) -> str:
-        """Escape the markdown underlines and spoilers."""
-        return re.sub(r"[_|]", lambda match: "\\" + match[0], content)
+        """Escape the markdown underlines and spoilers that aren't in codeblocks."""
+        markdown_regex = r"(?:`[^`\\]*(?:\\.[^`\\]*)*`)|([_|])"
+        markdown_outside_codeblocks_indices = [m.span()[0] for m in re.finditer(markdown_regex, content) if m.group(1)]
```

Comment:
Seems like I forgot about it, please use a named capture group instead of relying on indices to make the regex more resilient

Reply:
Addressed as part of the redesign :+1: